### ISSUE TITLE
PRC PRC_TYPE_GRAPH_TextureDefinition

### ISSUE THREAD

**Issue by mvrhel** [pdf-association/pdf-issues#412]
**PRC PRC_TYPE_GRAPH_TextureDefinition Table 96**
The definition of PRC PRC_TYPE_GRAPH_TextureDefinition includes two objects with Name transformation.   I do realize one is the texture transformation, but a spec like this should avoid name aliasing.      According to the table the texture transformation is of the type PRC_TYPE_GRAPH_TextureTransformation which makes sense.  The other one is of type Cartesian Transformation, which is not really a type name in the spec.  There is PRC_TYPE_MISC_CartesianTransformation.  Is that what this refers to?

**Comment by stuartgalt** [comment]
Yes the first transformation in that table is  PRC_TYPE_MISC_CartesianTransformation and the type column should be changed to reflect the actual type.

The last two lines of the table are "has_texture_transformation" and "texture_transformation" to avoid confusion with the cartesian transform in the table.

**Comment by petervwyatt** [comment]
3D PDF TWG agree
